src/components/AddBook.module.css

```diff
@@ -1,14 +1,47 @@
 .Books {
   padding: 1rem;
   border: 1px solid #cecece;
-  margin: 1rem;
+  margin: 2.313rem 6.25rem 0;
 }
 
 .BoxInput {
   display: flex;
-  gap: 1rem;
+  justify-content: space-between;
+  margin-top: 00.5rem;
 }
 
-.Books form span {
-  font-weight: 500;
+.adBook {
+  height: 1.5rem;
+  font-size: 1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.BoxInput input {
+  width: 29%;
+}
+
+.BoxInput input[type=text] {
+  padding: 0.5rem;
+}
+
+.BoxInput select {
+  padding: 0.5rem;
+}
+
+.BoxInput button {
+  width: 20%;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border-radius: 3px;
 }
```

Comment:
- [ ]  Please check the design provided to meet with this circular diagram as it  is shown on the design given

![image](https://user-images.githubusercontent.com/55635977/193407817-bb0c207c-a762-46fc-be43-5767f0770e96.png)